pincer_bot/cogs/info.py

```diff
@@ -92,5 +94,35 @@ async def pypi_dl_command(self):
             value=f'```py\n{rotated}\n```'
         )
 
+    @Client.event
+    async def on_message(self, message: UserMessage):
+        if "##" not in message.content:
```

Comment:
should we allow '#' :thinking:

Reply:
Honestly I'd allow it + make `##` not embed it, kinda like an escape sequence :thinking:

Reply:
Or even just use \# for that. Discord doesn't render the slash.